module/platform/darwin/HardwareIO/src/HardwareIO.cpp

```diff
@@ -191,17 +227,27 @@ namespace platform {
                     servo.presentSpeed    = Convert::servoSpeed(i, data.servos[i].presentSpeed);
                     servo.load            = Convert::servoLoad(i, data.servos[i].load);
 
+
                     // Diagnostic Information
                     servo.voltage     = Convert::voltage(data.servos[i].voltage);
                     servo.temperature = Convert::temperature(data.servos[i].temperature);
+
+                    if (servo.voltage <= maxVoltage) {
```

Comment:
Put a comment in here that you're clearing the over voltage flag if it's less than our expected voltage